source/index.rst

```diff
@@ -1,140 +1,141 @@
-Topics
```

Comment:
Is there a reason why there is a massive diff here?

Reply:
Oh! Honestly, I have no idea why. Maybe I mistakenly edited something without knowing